discord/ext/commands/errors.py

```diff
@@ -43,6 +43,17 @@
     'CommandOnCooldown',
     'MaxConcurrencyReached',
     'NotOwner',
+    'MessageNotFound',
+    'MemberNotFound',
+    'UserNotFound',
+    'ChannelNotFound',
+    'ChannelNotReadable',
+    'BadColourArgument',
+    'RoleNotFound',
+    'BadInviteArgument',
+    'EmojiNotFound',
+    'PartialEmojiConversionFailure',
+    'InvalidBoolean',
```

Comment:
```suggestion
    'BadBoolArgument',
```

Might as well be consistent with it.